exchange/utils.go

```diff
@@ -217,6 +218,12 @@ func getAuctionBidderRequests(req AuctionRequest,
 		return nil, []error{err}
 	}
 
+	var bidderExt map[string]json.RawMessage
+	bidderExt, err = adapters.ExtractBidderParams(req.BidRequest)
```

Comment:
Does `ExtractBidderParams` return the bidder ext? Looks like it returns the bidder params, and if so - consider renaming the variable here please.

Reply:
Fixed.